components/components.css

```diff
@@ -0,0 +1,848 @@
+  border: none;
+  border-radius: 5px;
+  font-weight: 700;
+  cursor: pointer;
+  font-family: 'Roboto Condensed', sans-serif;
+  margin-right: 0.5rem;
+}
+.btn__discord {
+  padding: 0.6rem;
+  background-color: #dc2626;
+  border: none;
+  border-radius: 5px;
+  font-weight: 700;
+  cursor: pointer;
+  font-family: 'Roboto Condensed', sans-serif;
+  margin-right: 0.5rem;
+}
+.btn__github {
+  padding: 0.6rem;
+  background-color: #dc2626;
+  border: none;
+  border-radius: 5px;
+  font-weight: 700;
+  cursor: pointer;
+  font-family: 'Roboto Condensed', sans-serif;
+  margin-right: 0.5rem;
+}
+.fab {
+  margin-right: 0.5rem;
+}
+/*cards*/
+.card__container {
+  margin-left: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2rem;
+}
+.card__container h1 {
+  padding-bottom: 1rem;
+}
+.card__container p {
+  padding-left: 1px;
+}
+.cards {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-left: 300px;
+}
+.card__border__wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  background-color: #f3f4f6;
+  box-shadow: 0 5px rgba(104, 102, 102, 0.2);
+  margin: 1rem;
+}
+.card__border {
+  border: 2px solid rgb(148, 148, 148);
+  border-radius: 5px;
+  width: 350px;
+  margin: auto;
+  padding: 1rem;
+  text-align: left;
+  margin-bottom: 0.5rem;
+  background-color: white;
+  margin: 1rem 0;
+}
+.card__title {
+  margin: 0.1rem 0;
+}
+.card__rating {
+  margin-right: 0.2rem;
+}
+.card__text {
+  margin-bottom: 0.3rem;
+  color: gray;
+}
+.card__old__price {
+  text-decoration: line-through;
+  color: gray;
+  font-size: 14px;
+  font-weight: lighter;
+  margin: 0 0.2rem;
+}
+.card__discount {
+  color: red;
+  font-size: 14px;
+  font-weight: 100;
+}
+.card__btn {
+  text-decoration: none;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  max-width: 100px;
+  justify-content: flex-start;
+  margin-top: 0.5rem;
+  padding: 0.6rem;
+  background-color: #dc2626;
+  border: none;
+  border-radius: 5px;
+  font-weight: 700;
+  cursor: pointer;
+  font-family: 'Roboto Condensed', sans-serif;
+  margin-right: 0.5rem;
+  text-align: center;
+}
+.card__borderless {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  background-color: #f3f4f6;
+  box-shadow: 0 5px rgba(104, 102, 102, 0.2);
+  margin: 1rem;
+}
+.card {
+  width: 350px;
+  margin: auto;
+  padding: 1rem;
+  text-align: left;
+  background-color: white;
+  margin: 1rem 0;
+}
+/*avatars*/
+.avatar__container {
+  margin-left: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 2rem;
+}
+.avatar__container h1 {
+  padding-bottom: 1rem;
+}
+.avatar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 1rem;
+  background-color: #f3f4f6;
+  box-shadow: 0 5px rgba(104, 102, 102, 0.2);

```

Comment:
You can make this rgba value as a CSS variable.

Reply:
Done with the changes suggested by you.